.github/workflows/ci.yml

```
@@ -26,19 +34,175 @@ jobs:
+      
   # OS 
-  # Ubuntu
-  build-ubuntu-latest:
+  # Ubuntu - manual build and test
+  # job build ubuntu latest manual
+  build-ubuntu-latest-manual:
+    name: '[Manual] Build Ubuntu latest'
+    needs: manual
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)